### PR TITLE
Store decrypted request method and content data

### PR DESCRIPTION
**Pull request by bumi**
this allows use to create an event log in the future and list the actual requests per app connection

### CODE REVIEW

File: service.go [hunk 1]
```diff
@@ -541,6 +541,10 @@ func (svc *Service) HandleEvent(ctx context.Context, sub *nostr.Subscription, ev
 		return
 	}
 
+	requestEvent.Method = nip47Request.Method
+	requestEvent.ContentData = payload
```
Comment (rolznz) [inline service.go:545]: should the `ContentData` be `nip47Request.params`?

```
{
    "method": "pay_invoice",
    "params": {
        "invoice": "lnbc50n1...", // bolt11 invoice
        "amount": 123, // invoice amount in msats, optional
    }
}
```

And should we call it `params` to be consistent with the spec?

Comment (bumi) [reply service.go:545]: do you think that's better? 
then I have to JSON dump the params here.

Comment (rolznz) [reply service.go:545]: True. Maybe the way you did it is better.

Comment (rolznz) [reply service.go:545]: Do we still need to store the `Content` then? or we can remove this along with the content in the response event?

**Review by rolznz — APPROVED** [review]
utACK

**Pull request merged**